frontends/web/src/api/market.ts

```diff
@@ -17,44 +17,44 @@
 import { AccountCode } from './account';
 import { apiGet, apiPost } from '@/utils/request';
 
-export const getExchangeRegionCodes = (): Promise<string[]> => {
-  return apiGet('exchange/region-codes');
+export const getMarketRegionCodes = (): Promise<string[]> => {
+  return apiGet('market/region-codes');
 };
 
-export type TPaymentMethod = 'card' | 'bank-transfer' | 'bancontact' | 'sofort' | 'spend';
+export type TPaymentMethod = 'card' | 'bank-transfer' | 'bancontact' | 'sofort' | 'spend'; // FIXME
```

Comment:
Maybe change fixme to TODO with a comment what has to be done?

Reply:
Please no TODOs in code, they tend to stick around forever and become not actionable due to the lack of context. Please open issues for follow-ups that contain all the context needed.